### PR TITLE
Support converting TENDL 2019 library

### PR DESCRIPTION
**Pull request by paulromano**
Addresses #15. This one went really smoothly. I didn't need to make any bug fixes in the ACE files distributed by TENDL (for once!). @Shimwell, mind reviewing?

### CODE REVIEW

**Comment by Shimwell** [comment]
Hi @paulromano sorry I just noticed this, I shall review it today / tomorrow.

**Comment by Shimwell** [comment]
This is a great use of those dictionary descriptions that we investing in a while back. It makes adding new releases straight forward to implement and straight forward to check.

I have used your updated script to generate the new tendl 2019

I noticed that they have added a few more isotopes compared to 2017 (557) compared to 2019 (631)

Looking at the two cross_sections.xml generated the new TENDL 2019 brings in some extra isotopes for Ag, Al As, Bi, Br, Co, Cu, Dy, Er, Eu, Fe, Ga, Gd, Ho, I, Ir, La, Lu Mn, Nb, Ni, Os, P, Rb, Re, Rh, Sb, Sc, Sn, Sr, Ta, Tc, Ti, V, W, Xe, Y, Zr

This is a very welcome addition to the repo. As the next step this script can be upgraded to use the standard inputs and use of Pathlib that some of the other new scripts use, but I am aware that these particular features have been contracted out and others are planning to do that.

I have just one potential suggested change, the compressed file size on my system shows up as 2.3GB so only a minor difference there. The uncompressed file size of both the ./tendl-2019-ace and the ./tendl-2019-hdf5 folder shows up as (2.4GB + 7.7GB or 10.1GB in total). These numbers are slightly different to the numbers in the script. Not really an issue but for your consideration.

File: convert_tendl.py [hunk 1]
```diff
@@ -63,6 +63,14 @@ class CustomFormatter(argparse.ArgumentDefaultsHelpFormatter,
         'metastables': os.path.join(ace_files_dir, 'ace-17', '*m'),
         'compressed_file_size': '2.1 GB',
         'uncompressed_file_size': '14 GB'
+    },
+    '2019': {
+        'base_url': 'https://tendl.web.psi.ch/tendl_2019/tar_files/',
+        'files': ['tendl19c.tar.bz2'],
+        'neutron_files': os.path.join(ace_files_dir, 'tendl19c', '*'),
+        'metastables': os.path.join(ace_files_dir, 'tendl19c', '*m'),
+        'compressed_file_size': '2.2 GB',
```
Comment (Shimwell) [inline convert_tendl.py]: perhaps 2.3 GB

File: convert_tendl.py [hunk 2]
```diff
@@ -63,6 +63,14 @@ class CustomFormatter(argparse.ArgumentDefaultsHelpFormatter,
...
+        'uncompressed_file_size': '6.8 GB'
```
Comment (Shimwell) [inline convert_tendl.py]: Using the terminal command "du -h" I get a larger file size here
2.4G	./tendl-2019-hdf5
7.7G	./tendl-2019-ace

File: convert_tendl.py (hunk 1 above)
Comment (Shimwell) [inline convert_tendl.py]: I get a very slight larger file size here
2.3G Feb  6 08:31 tendl19c.tar.bz2

**Comment by paulromano** [comment]
Thanks @Shimwell for the review; just updated those file sizes

**Comment by Shimwell** [comment]
Thanks Paul, let's merge this

**Pull request merged**